sdk/azcore/log.go

```diff
@@ -11,6 +11,7 @@ const (
 	LogError        LogClassification = "Error"
 	LogRequest      LogClassification = "Request"
 	LogResponse     LogClassification = "Response"
+	LogRetry        LogClassification = "Retry"
```

Comment:
This si really just for debugging the retry logic, right? If so, a comment to that effect would be nice to have here.

Reply:
Will rename to `LogRetryPolicy`.  All of these enums need comments :(